### PR TITLE
Add pygraphml

### PR DESCRIPTION
**Pull request by hadim**
https://github.com/hadim/pygraphml

### CODE REVIEW

**Comment by hadim** [comment]
Up ?

(sorry I don't know the usual delay to get reviewed here)

File: recipes/pygraphml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - pygraphml
+
+about:
+  home: https://github.com/hadim/pygraphml/
+  license: BSD License
```
Comment (ocefpaf) [inline recipes/pygraphml/meta.yaml]: [It looks](https://github.com/hadim/pygraphml/blob/master/LICENSE) like a `BSD 3-Clause`. Can you update the license version here before we merge this?

File: recipes/pygraphml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install
```
Comment (ocefpaf) [inline recipes/pygraphml/meta.yaml]: `pygraphml` seems to use `setuptools` so you can install it with the that preferred command:

`python setup.py install --single-version-externally-managed --record record.txt`

**Comment by hadim** [comment]
Done @ocefpaf . Thank you for the review.

**Pull request merged**